mm/mm_heap/mm_free.c

```diff
@@ -78,34 +78,37 @@ static void add_delaylist(FAR struct mm_heap_s *heap, FAR void *mem)
  *
  ****************************************************************************/
 
-void mm_delayfree(FAR struct mm_heap_s *heap, FAR void *mem, bool delay)
+void mm_delayfree(FAR struct mm_heap_s *heap, FAR void *mem,
```

Comment:
If you step back and look at this code. mm_delayfree does a optional delay and optional clean  based on parameters so the naming just adds confusion.